packages/coinstac-ui/app/render/utils/helpers.js

```diff
@@ -1,9 +1,37 @@
 import { indexOf } from 'lodash';
 
-// eslint-disable-next-line import/prefer-default-export
-export const isPipelineOwner = (permissions, owningConsortium) => {
+const COMP_INPUT_NEED_USER_DATA = [
+  'csv',
+  'freesurfer',
+  'files',
+];
+
+export function isPipelineOwner(permissions, owningConsortium) {
   return indexOf(permissions.consortia[owningConsortium], 'owner') !== -1;
-};
+}
+
+export function pipelineNeedsDataMapping(pipeline) {
+  if (!pipeline || !pipeline.steps) {
+    return false;
+  }
+
+  let needsDataMapping = false;
+
+  pipeline.steps.forEach((step) => {
+    const inputMapSchemaKeys = Object.keys(step.inputMap);
+
+    inputMapSchemaKeys.forEach((inputSchemaKey) => {
+      const compInputField = step.computations[0].computation.input[inputSchemaKey];
+      const compInputFieldType = compInputField.type.toLowerCase();
```

Comment:
?

Reply:
why is this lower casing?

Reply:
Just to make sure the compare works. But this was removed anyway.